package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "todo-list1",
+  "version": "1.0.0",
+  "description": "> Todo list is a web app help you to management well your important tasks",
+  "main": "webpack.config.js",
+  "scripts": {
+    "start": "webpack",
+    "serve": "webpack serve --open"
+  },
```

Comment:
- [ ] When you run the command `npm run build` it suppose to generate a HTML and JS files in your `dest` folder and when you run `npm start` the server suppose to work and show the generated files in your browser That not happening because you changed this script commands in a wrong way so Please fix that.
[ Hint ]: Kindly go back again to the exercise before this project and follow the instructions carefully and if you installed everything the right way  your script command will be like that:

```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Reply:
Hello @Omar-Muhamad 
Did you try to run with npm run serve

Reply:
Hello @Omar-Muhamad 
I'm done

Reply:
sorry i just saw that give me 10 min and I will check

Reply:
@Omar-Muhamad  All is good now